pandas/tests/io/test_stata.py

```diff
@@ -1958,6 +1958,16 @@ def test_backward_compat(version, datapath):
     tm.assert_frame_equal(old_dta, expected, check_dtype=False)
 
 
+@pytest.mark.parametrize("version", [105, 108, 111, 113, 114, 118])
+def test_bigendian(version, datapath):
+    data_base = datapath("io", "data", "stata")
+    ref = os.path.join(data_base, f"stata-compat-{version}.dta")
+    big = os.path.join(data_base, f"stata-compat-be-{version}.dta")
```

Comment:
```suggestion
    ref = datapath("io", "data", "stata", f"stata-compat-{version}.dta")
    big = datapath("io", "data", "stata", f"stata-compat-be-{version}.dta")
```

Reply:
I originally modelled the test function on `test_backward_compat`, but am happy with this suggested change.